app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t(".success")
+      redirect_to @user
+    else
+      render "new"
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
```

Comment:
chỗ này em có thể bỏ ngoặc khi gọi method ở cuối
```suggestion
    params.require(:user).permit :name, :email, :password,
                                 :password_confirmation

```